src/api/webhook/types.ts

```diff
@@ -0,0 +1,42 @@
+/**
+ * LINKURIOUS CONFIDENTIAL
+ * Copyright Linkurious SAS 2012 - 2024
+ *
+ * - Created on 2024-02-05.
+ */
+import {PaginationClause, PersistedItem} from '../commonTypes';
+
+export interface CreateWebhookParams {
+  url: string;
+  secret: string;
+  events?: WebhookEvent[];
+}
+
+export interface DeleteWebhookParams {
+  webhookId: number;
+}
+
+export type PingWebhookParams = DeleteWebhookParams;
+
+export interface GetWebhookDeliveriesParams extends DeleteWebhookParams, PaginationClause {}
```

Comment:
The fact that multiple interfaces that have nothing to with delete extend the delete params looks weird in my opinion, it makes sense programatically but does not make sense functionally.

I think the cost to make functionally coherent is pretty low.
e.g:
```ts
interface ExistingWebhookParams {
  webhookId: number;
}

export interface DeleteWebhookParams extends ExistingWebhookParams {}

export interface PingWebhookParams extends ExistingWebhookParams {}

export interface GetWebhookDeliveriesParams extends ExistingWebhookParams, PaginationClause {}

```

Bonus point: Adding additional params to `DeleteWebhookParams` will not affect any other interface.

Reply:
Good idea, it's done :+1: